### ISSUE TITLE
KK_MapDecal_Miraflores_7 needs adjustment

### ISSUE THREAD

**Issue by caerfinon** [caerfinon/PanoceanCanal#3]
Ground clips over edge of cliff when graphics set to high. 
Can be fixed by adjusting LAT/LON until clip goes away

**Comment by caerfinon** [comment]
In the file \GameData\PanoceanCanal\Miraflores\KK_MapDecal_Miraflores_7.cfg

Original Values are
```
	Latitude = 46.862942881249921
	Longitude = 87.787399178748686
	AbsolutOffset = -99.7679977
```

Corrected values are 

```
	Latitude = 46.861774076025029
	Longitude = 87.782651727987343
	AbsolutOffset = -99.313652
```

Issue only appears on some PCs depending on thier Graphics settings

**Comment by caerfinon** [comment]
- Adjusted KK_MapDecal_Miraflores_7 to fix video clipping
- Changed Gatun base to correct ground contact issues. 
- Closed with release v1.0.1
